### PR TITLE
Updated ACME grammar from GoDot

### PR DESCRIPTION
**Pull request by MatthewCallis**

### CODE REVIEW

File: grammars/acme.cson [hunk 1]
```diff
@@ -33,74 +33,99 @@ repository:
...
+        match:  '(?i:while|until|else)\\b'
         name:   'keyword.control.conditional.acme'
       }
-      # Operators
-      {
-        match:  '!|\\+|\\-|\\/|\\*|<<|>>|&|\\||\\^|=|<|>|\\:|\\|\\|'
-        name:   'keyword.operator.acme'
-      }
-      # Operators II
+      # Operators (don't work)
```
Comment (georgjz) [inline grammars/acme.cson]: What is the trouble with this? When I uncomment it, it works fine, as do Operators II. 

Have you checked if your syntax theme in Atom highlights operators? I found that some syntax themes _simply do not highlight certain scopes_. I had that problem when writing a grammar for 6809 assembly, my grammar code was correct but my syntax theme simply didn't include code for highlighting certain scopes. Switching between several syntax themes sometimes reveals that problem.

EDIT: I use Monokai Seti as syntax theme

File: grammars/acme.cson [hunk 2]
```diff
@@ -33,74 +33,99 @@ repository:
   # symbols
   symbols:
     patterns: [
+#     Labels (at beginning of a line)
+      {
+        match: '^[A-Za-z][A-Za-z0-9_:]*'
```
Comment (georgjz) [inline grammars/acme.cson]: I suggest changing this to 
```
match: '^[A-Za-z\\._][A-Za-z0-9_:]*'
```
Global labels can start with a underscore, local labels with a colon.

**Review by georgjz — COMMENTED** [review]
I added two suggestions.

File: grammars/acme.cson (hunk 2 above)
Comment (MatthewCallis) [reply grammars/acme.cson]: Thanks, updated! I'm just doing this PR on GoDot's behalf.

File: grammars/acme.cson (hunk 1 above)
Comment (MatthewCallis) [reply grammars/acme.cson]: I hadn't tried, I will uncomment them.

**Comment by MatthewCallis** [comment]
I fixed the items you pointed out, I trusted that GoDot had tested this but the syntax highlighting issue makes sense, thank you!

**Pull request merged**